cmd/pomerium/main.go

```diff
@@ -181,3 +190,25 @@ func parseOptions(configFile string) (*config.Options, error) {
 	}
 	return o, nil
 }
+
+func handleConfigUpdate(opt *config.Options, services []config.OptionsUpdater) *config.Options {
+	newOpt, err := parseOptions(*configFile)
+	log.Debug().Msgf("cmd/pomerium: old config = %s, new config = %s", opt.Checksum(), newOpt.Checksum())
```

Comment:
nit: thoughts on using something like the following log invocation? ( I believe this is 0 allocations too)
```go
log.Debug().Str("old-config", opt.Checksum()).Str("new-config", newOpt.Checksum().Msg("cmd/pomerium: config update")
```

Reply:
I like it.  Wasn't familiar with zerolog.  This makes sense after looking at their docs.  👍

Reply:
Updated.  Please review the new logging approach.